source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,1422 @@
+{
```

Comment:
<p>This would be better to use the PipInstall plugin as it would also get run on new workers if they get scaled up in the future.</p><p> </p><pre class="ql-syntax" spellcheck="false">from dask.distributed import PipInstall

client.register_worker_plugin(PipInstall(packages=["adlfs"]))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>